extensions/control-plane/api/management-api/asset-api/src/main/java/org/eclipse/edc/connector/api/management/asset/model/AssetUpdateDto.java

```diff
@@ -0,0 +1,69 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.connector.api.management.asset.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import jakarta.validation.constraints.AssertTrue;
+import jakarta.validation.constraints.NotNull;
+
+import java.util.Map;
+
+@JsonDeserialize(builder = AssetUpdateDto.Builder.class)
+public class AssetUpdateDto {
```

Comment:
Couldn't this be used as superclass for `AssetRequestDto`? Maybe it's just useless hierarchy, but it would remove some duplication

Reply:
so, we'd have to have an `AssetDto` (superclass, just has the `Map`), and derived from that the `AssetRequestDto`, that adds the `id`, and potentially also an `AssetUpdateDto`, that is just a marker and adds nothing. The latter is not needed, strictly speaking. WDYT?

Reply:
`AssetDto` does not exist... otherwise it could have been the right object to do, in fact now there's no hierarchy on `AssetRequestDto`, in fact it represent a creation request, that at the end is an update request plus the id.
Could be weird but maybe it's just a naming issue...

`AssetCreateRequestDto` that's subclass of
`AssetUpdateRequestDto`

sounds better to me

Reply:
> AssetDto does not exist...
i know, i meant that would probably be the base class's name :)

Let me try and refactor these DTOs a bit, we'll see how it turns out

Reply:
check https://github.com/eclipse-edc/Connector/pull/2516/commits/9b352742ec353f9fd4b9d7c2ed062b2e5e16c710

Reply:
@paullatzelsperger looks good!